tests/rsync/rsync_spec.lua

```diff
@@ -27,15 +27,35 @@ describe("rsync", function()
             assert.equals(err, "")
         end)
 
-        it("RsyncProjectConfig", function()
+        it("RsyncProjectConfig show", function()
```

Comment:
```suggestion
        it("RsyncProjectConfig", function()
```
No arguments should still be valid just as before.